apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/ValueParserImpl.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c)  2018-2019. Apollo Foundation.
+ */
+
+package com.apollocurrency.aplwallet.apl.core.shard.helper;
+
+import com.apollocurrency.aplwallet.apl.core.shard.helper.csv.ValueParser;
+import com.apollocurrency.aplwallet.apl.util.StringUtils;
+
+import javax.inject.Singleton;
+import java.util.Base64;
+
+@Singleton
+public class ValueParserImpl implements ValueParser {
+    @Override
+    public String parseStringObject(Object data) {
+        String stringObject = (String)data;
```

Comment:
Check for NULL pls.

Reply:
Please, consider this code:
final String stringObject = `Optional.ofNullable(data).map(String.class::cast).orElseThrow(() -> new IllegalArgumentException(""));`

Reply:
done